modules/tabs-v2/widgets/tabs-v2.php

```diff
@@ -369,12 +414,25 @@ protected function render() {
 				'tabindex' => 1 === $tab_count ? '0' : '-1',
 				'aria-controls' => 'elementor-tab-content-' . $id_int . $tab_count,
 				'aria-expanded' => 'false',
+				'id' => $tab_id,
```

Comment:
It's a duplication of the `id` (desktop/mobile)

Reply:
2nd item changed to {element-id}-container